CROSS_CHAIN_RECOVERY.md

```diff
@@ -0,0 +1,33 @@
+# Cross Chain Recovery Instructions
+
+In a cross-chain transaction, you attempt to send funds to an address, but accidentally perform the transaction on the wrong chain.
+
+For example, an exchange may ask you to deposit Bitcoin Cash to the Bitcoin Cash address 36jfYqR3J29nAEfYqgksVJ8iVrvLmEb51v. Y  ou misread or don't understand the instructions, and instead send Bitcoin to the address. This is a BTC→BCH cross-chain transaction.
```

Comment:
If you sent Bitcoin to the address by mistake, this would be a BTC->BCH cross-chain transaction

Reply:
Corrected